makefile

```diff
@@ -1,21 +1,25 @@
 NUM_NODES = 10
 WORLD_SIZE = 1000
 
-COMMON_CFLAGS = -g -std=gnu99 -O2 -mcpu=native -fomit-frame-pointer -Wall -Wextra
+COMMON_CFLAGS = -std=gnu99 -O2 -march=native -fomit-frame-pointer -Wall -Wextra
```

Comment:
It's not documented to work like that in the [GCC ARM docs](https://gcc.gnu.org/onlinedocs/gcc/ARM-Options.html). Moreover, the names accepted by the -march option are architectures, while the names accepted by mtune are specific cores.